### DISCUSSION TITLE
Youtube seems to have moved livestreams (and the resulting vods) to a live section. As a result these videos do not show up in FreeTube.

### DISCUSSION THREAD

**Discussion by ghost** [FreeTubeApp/FreeTube#2821] [General]
I don't visit the Youtube site often so I don't know when this was implemented (I searched the issues and discussions here but found nothing) but Youtube seems to have moved livestreams (and the resulting vods) from videos to its own section labeled 'live'. As a result these videos now do not show up in FreeTube. 

[Example](https://www.youtube.com/channel/UCMwGHR0BTZuLsmjY_NT5Pwg/streams). 

Is this something you're aware of or is it brand new? Should I open a feature request?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @voidscuttle just before we released, YouTube started to shift things around again. There are now independent tabs on the channel page for shorts and livestreams. You are also right about it not showing up in the subscription tab. There are already a few fixes implemented for this to show up in your subscription tab again. 

So in summary we are aware of this and we are working on fixing this :)
